src/components/InfoHouseContainer/InfoHouseContainer.scss

```diff
@@ -0,0 +1,3 @@
+.img-style {
+    height: 100px;
+}
```

Comment:
@berni23 Debería haber siempre una línea en blanco al final de cada fichero. Si os fijáis en el icono de la línea 3 os marca eso.